homeassistant/components/soundtouch/media_player.py

```diff
@@ -350,6 +352,16 @@ def play_media(self, media_type, media_id, **kwargs):
             else:
                 _LOGGER.warning("Unable to find preset with id %s", media_id)
 
+    def select_source(self, source):
+        if source == "AUX":
```

Comment:
A docstring is mandatory, just the description of the parameters is unnecessary. If you add the docstring back the CI validate error will be solved.

Reply:
Yep, saw that in the CI result.
I just activated the Flake8 git commit hook, so that shouldn't happen anymore